index.js

```diff
@@ -27,10 +28,20 @@ function fastifyView (fastify, opts, next) {
   const includeViewExtension = opts.includeViewExtension || false
   const prod = typeof opts.production === 'boolean' ? opts.production : process.env.NODE_ENV === 'production'
   const defaultCtx = opts.defaultContext || {}
+  const layoutFileName = opts.layout
+
+  if (layoutFileName) {
+    try {
+      readFileSync(join(templatesDir, getPage(layoutFileName, 'hbs')))
+    } catch (e) {
+      next(e)
```

Comment:
this should `return`. Can you add a unit test for this case?

Reply:
Thanks for pointing out this should return.

Regarding the test, would something like this work? https://github.com/fastify/point-of-view/pull/136/files#diff-c1129c8b045390789fa8ff62f2c6b4a9R111